public/app/features/logs/components/LogRows.tsx

```diff
@@ -56,6 +56,29 @@ interface State {
   renderAll: boolean;
 }
 
+// this class generates react-keys that are guaranteed to be unique.
```

Comment:
I would 100% move this to its own module. Not only for organization, but to allow it to be used in other places.

Reply:
sure! moved it to a separate file in the logs "area".